english/make_changes_GitHub/clone_source/stuck.md

```diff
@@ -0,0 +1,5 @@
+You open up the command line and use Git to clone the repo to your computer. You begin making changes to the story to get yourself out of this awful world. When you finish your changes, nothing happened.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.